tests/Makefile

```diff
@@ -14,9 +14,6 @@ PYTEST_ARGS :=
 DOTENV := $(PWD)/.env
 
 BUILDDIR := build
-# These are for dumping logs, but they're really long so store them as variables
-E2E_LOG_PODS := {range .items[?(@.metadata.annotations.helm\.sh\/hook)]}{@.metadata.name}{"\n"}{end}
```

Comment:
We don't need this line anymore?

Reply:
it was used in the 2nd log dump loop that I removed here https://github.com/scality/Zenko/commit/1674be4d387830e21eb740844b396f4753123b2c